<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="3" total="4" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Login]]>
    </line>
    <line>
      <![CDATA[editProduct]]>
    </line>
    <line>
      <![CDATA[Logout]]>
    </line>
  </reporter-output>
  <suite name="Failed suite [Failed suite [Default suite]]" duration-ms="26" started-at="2018-03-01T02:47:22Z" finished-at="2018-03-01T02:47:22Z">
    <groups>
      <group name="product">
        <method signature="GroupingConceptinTestNG.editProduct()[pri:0, instance:qsp.GroupingConceptinTestNG@50c87b21]" name="editProduct" class="qsp.GroupingConceptinTestNG"/>
      </group> <!-- product -->
      <group name="smoke">
        <method signature="GroupingConceptinTestNG.editProduct()[pri:0, instance:qsp.GroupingConceptinTestNG@50c87b21]" name="editProduct" class="qsp.GroupingConceptinTestNG"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Default test(failed)(failed)" duration-ms="26" started-at="2018-03-01T02:47:22Z" finished-at="2018-03-01T02:47:22Z">
      <class name="qsp.GroupingConceptinTestNG">
        <test-method status="PASS" signature="login()[pri:0, instance:qsp.GroupingConceptinTestNG@50c87b21]" name="login" is-config="true" duration-ms="3" started-at="2018-03-01T08:17:22Z" finished-at="2018-03-01T08:17:22Z">
          <reporter-output>
            <line>
              <![CDATA[Login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="editProduct()[pri:0, instance:qsp.GroupingConceptinTestNG@50c87b21]" name="editProduct" duration-ms="5" started-at="2018-03-01T08:17:22Z" finished-at="2018-03-01T08:17:22Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[It failed again....]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: It failed again....
	at org.testng.Assert.fail(Assert.java:94)
	at qsp.GroupingConceptinTestNG.editProduct(GroupingConceptinTestNG.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[editProduct]]>
            </line>
          </reporter-output>
        </test-method> <!-- editProduct -->
        <test-method status="PASS" signature="logout()[pri:0, instance:qsp.GroupingConceptinTestNG@50c87b21]" name="logout" is-config="true" duration-ms="1" started-at="2018-03-01T08:17:22Z" finished-at="2018-03-01T08:17:22Z">
          <reporter-output>
            <line>
              <![CDATA[Logout]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- qsp.GroupingConceptinTestNG -->
    </test> <!-- Default test(failed)(failed) -->
  </suite> <!-- Failed suite [Failed suite [Default suite]] -->
</testng-results>
